src/MetadataService.ts

```diff
@@ -17,6 +17,7 @@ export class MetadataService {
     private _metadataUrl: string;
     private _signingKeys: SigningKey[] | null = null;
     private _metadata: Partial<OidcMetadata> | null = null;
+    private fetchRequestCredentials: RequestCredentials | undefined;
```

Comment:
`fetchRequestCredentials` -> `_fetchRequestCredentials` (underscore)

Reply:
and align type and default value to others: 
`private _fetchRequestCredentials: RequestCredentials | null = null;`

Reply:
Default value should be undefined then the credentials property will skipped if the _fetchRequestCredentials isn't set.

Reply:
Ex here: https://github.com/authts/oidc-client-ts/pull/778/files#diff-dc78e3cc058d3fd2461eec10c5b75ee1c28394a27aadf3ab8a8fe3294547b9e3R58

Reply:
Renaming done.

Reply:
so only the rename (fetchRequestCredentials -> _fetchRequestCredentials) here...